src/client/components/SupplierEditor/SupplierEditor.react.js

```diff
@@ -193,7 +193,7 @@ class SupplierEditor extends Component {
 
     if (hasErrors) {
       return (
-        <div>{ this.context.i18n.getMessage('SupplierEditor.Messages.unableToRender') }</div>
+        <div>{ this.context.i18n.getMessage('SupplierEditor.Messages.unableToRender')  } <a href="/bnp/"> CLick here</a> </div>
```

Comment:
`Click here` should changed to `Register` (de: `Registrieren`). Also, the a tag should have css classes `btn btn-link`